spinalcordtoolbox/reports/assets/index.html

```diff
@@ -20,6 +20,11 @@
       document.getElementById("mosaic-container").style.marginBottom=image_height+"px";
     }
   </script>
+  <style>
+      .fixed-table-container{
+          height: fit-content;
+      }
+  </style>
```

Comment:
Aha. I think the issue is because `fixed-table-container:: height` is defined in two separate places. Right here, but also here:

https://github.com/neuropoly/spinalcordtoolbox/blob/0211c4adeb951b3bc4d3e44d9e560b4710ef5e1d/spinalcordtoolbox/reports/assets/_assets/css/style.css#L71-L74

I've modified `style.css` and my page loads almost as expected:

![Screenshot from 2021-04-20 10-26-16](https://user-images.githubusercontent.com/16181459/115413086-e3a68300-a1c2-11eb-910d-f5e0da7f1991.png)

The table styling is still a little odd, though. The borders are missing, and when I first highlight the image row, the header row also gets highlighted.

Reply:
Usually, when the page is loaded, the browser takes the last definition of the style for a specific element (since it loads everything sequentially).

Reply:
That makes sense. I wonder why changing it fixed things for me, then. :thinking: